tests/test_report_generator.py

```diff
@@ -13,16 +15,40 @@
 # cisagov Libraries
 from tpt_reports import report_generator
 
+FILE_PATH = "./tests/data"
```

Comment:
You should use the full path to the test JSON file, as is done in the other test:
https://github.com/cisagov/tpt-reports/blob/807cbe870fddb0b63ea9c89cf72f5d1723510be4/tests/test_tpt_reports.py#L27

If you need to get the directory that the test file resides in, you can use [`os.path.dirname()`](https://docs.python.org/3/library/os.path.html#os.path.dirname).

Reply:
Resolved in [8c1eb5f](https://github.com/cisagov/tpt-reports/pull/56/commits/8c1eb5fcd8768027c58a5a223491268a233f4cc2)